flink-python/pyflink/table/table_result.py

```diff
@@ -48,6 +51,22 @@ def get_job_client(self):
         else:
             return None
 
+    def wait(self, timeout_ms=None):
```

Comment:
call it `await` to be consistent. how are the exceptions handled in Python?

Reply:
`await` is a keyword in python.  
just like `as` method in `Table`, python `Table` uses `alias`.

now flink-python does not defined Exceptions, java exception will be thrown directly. example: python `Catalog` class